frontend/src/shared/modules/filters/components/filterTypes/BooleanFilter.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <div v-if="form">
-    <cr-filter-include-switch v-if="!props.hideIncludeSwitch" v-model="form.include" />
```

Comment:
I think here you missed to delete the component import?
Also to remove `include` from `defaultForm`